Launching other Programs/main.py

```diff
@@ -0,0 +1,6 @@
+#Python script to launch anyother program
+import os
+import subprocess
+
```

Comment:
Kindly add some choice to users like calculator, notepad, Wordpad.

Reply:
Choices for the other programs should be given using user input or argument variables or predefined choices in the script? @yashaswibiyahut

Reply:
user input